### PR TITLE
Update to modules

### PR DESCRIPTION
**Pull request by olekristian**
Minor adjustments to language, some translations have also been made.
Please check line 69 of missionHelper/i18n\nb_NO.json - is this the correct fix when the vehicle is not displayed properly? http://prntscr.com/vhls72

### CODE REVIEW

File: src/modules/missionHelper/i18n/nb_NO.json [hunk 1]
```diff
@@ -60,34 +60,35 @@
...
 		"multifunctionals": {
 			"battalion_chief_vehicles": {
 				"additional_text": "",
 				"reduce_from": "mobile_command_vehicles"
 			},
 			"heavy_rescue_vehicles": {
-				"additional_text": "thereof at least. {n} Rescue Engines",
-				"reduce_from": "firetrucks"
+				"additional_text": "hvorav hvertfall {n} er redningsbiler.",
+				"reduce_from": "mannskapsbiler"
```
Comment (jxn-30) [inline src/modules/missionHelper/i18n/nb_NO.json:78]: Value needs to stay the key. In that case "firetrucks".

File: src/modules/missionHelper/i18n/nb_NO.json [hunk 2]
```diff
@@ -60,34 +60,35 @@
...
 			"platform_trucks": "Stigebil | Stigebiler",
-			"police_cars": "Patruljebil | Patriljebiler",
+			"police_cars": "Patruljebil | Patruljebiler",
 			"police_helicopters": "Politihelikopter | Politihelikoptre",
 			"sheriff": "Utrykningspolitibil | Utrykningspolitibiler",
 			"swat_armored_vehicle": "Delta kjøretøy | Delta kjøretøy",
 			"swat_suv": "Panstret kjøretøy | Pansdrede kjøretøy",
-			"water_tankers": "Tankbil | Tankbiler"
+			"water_tankers": "Tankbil | Tankbiler",
+			"traffic_car": "Utrykningspolitibil"
```
Comment (jxn-30) [inline src/modules/missionHelper/i18n/nb_NO.json:69]: Do you want to add a pluralized value?

**Review by jxn-30 — CHANGES_REQUESTED** [review]
Yes that's how to add new vehicle keys 👍

File: src/modules/missionHelper/i18n/nb_NO.json (hunk 2 above)
Comment (olekristian) [reply src/modules/missionHelper/i18n/nb_NO.json:69]: I'll add this to the finished file 👍 thanks for the heads up

**Comment by jxn-30** [comment]
@olekristian can we merge that or do you want to create further commits to this PR before merge?

**Comment by olekristian** [comment]
I'll just create a new pull request later!

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**